build.gradle

```
@@ -25,6 +32,13 @@ allprojects {
     }
 
     repositories {
+        maven {
```

According to the [doc](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-gradle-registry#using-a-published-package), it's common practice to:
1. provide a property alternative to env vars
2. do not hardcode username
3. name env variables to be more generic

```suggestion
        maven {
            url = uri("https://maven.pkg.github.com/epam/jclouds")
            credentials {
                username = project.findProperty("gpr.user") ?: System.getenv("USERNAME")
                password = project.findProperty("gpr.key") ?: System.getenv("TOKEN")
            }
        }
```

---

Done